doc/api/buffer.md

```diff
@@ -1264,7 +1264,7 @@ console.log(buf.lastIndexOf('buffer', 4));
 const utf16Buffer = Buffer.from('\u039a\u0391\u03a3\u03a3\u0395', 'ucs2');
 
 // Prints: 6
-console.log(utf16Buffer.lastIndexOf('\u03a3', null, 'ucs2'));
+console.log(utf16Buffer.lastIndexOf('\u03a3', buf.length, 'ucs2'));
```

Comment:
`null` didn't work? It looks like the example was trying to show how the second argument has a sensible default, and you have just provided the default value explicitly.